frontend/src/components/Common/ReportDialog/reportDoc.tsx

```diff
@@ -11,20 +12,33 @@ import { Theme } from '@material-ui/core';
 import { TableRow as AnalysisTableRow } from 'context/analysisResultStateSlice';
 import { getLegendItemLabel } from 'components/MapView/utils';
 import { LegendDefinition, ReportType } from 'config/types';
-import { TFunction } from 'utils/data-utils';
 import { Column } from 'utils/analysis-utils';
+import RobotoFont from 'fonts/Roboto-Medium.ttf';
+import KhmerFont from 'fonts/Khmer-Regular.ttf';
+import { useSafeTranslation } from 'i18n';
 import { PDFLegendDefinition } from './types';
 import ReportDocLegend from './ReportDocLegend';
 import ReportDocTable from './ReportDocTable';
 
-const makeStyles = (theme: Theme) =>
+// Register Fonts
+Font.register({
+  family: 'Roboto',
+  src: RobotoFont,
+});
+
+Font.register({
+  family: 'Khmer',
+  src: KhmerFont,
+});
+
+const makeStyles = (theme: Theme, selectedLanguage: string) =>
   StyleSheet.create({
     page: {
       flexDirection: 'column',
       backgroundColor: '#FFF',
       paddingBottom: 25,
       paddingTop: '1vh',
-      fontFamily: 'Roboto',
+      fontFamily: selectedLanguage === 'kh' ? 'Khmer' : 'Roboto',
```

Comment:
Let's create a small utility function for mapping. Either as a map or a switch maybe?

Reply:
on it

Reply:
Done